week7/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import logoImg from "../assets/Linkbrary.png";
+import "./Header.css";
+import Folder from "./Folder.js";
+import { useMediaQuery } from "react-responsive";
+const profileAccount = ({ user }) => {
+  if (!user) {
+    return false;
+  }
+  const { email, profileImageSource } = user;
+  return (
+    <div className="profile-info">
+      <img className="logo" src={profileImageSource} alt="프로필 이미지" />
```

Comment:
여기는 `alt` 속성의 값으로  프로필 이미지가 아니라 로고가 더 적당합니다.

```suggestion
      <img className="logo" src={profileImageSource} alt="코드잇 로고" />
```